Regression/WarpX-tests.ini

```diff
@@ -256,7 +256,7 @@ numthreads = 1
 compileTest = 0
 doVis = 0
 compareParticles = 1
-runtime_params = warpx.do_nodal=1
+runtime_params = warpx.do_nodal=1 algo.current_deposition=direct
```

Comment:
This also corrects the test for nodal 2d Langmuir: the test was using the esirkepov deposition, which is not valid for a nodal grid.